testground/benchmark/benchmark/stateless.py

```diff
@@ -225,7 +235,8 @@ def do_run(home: str, cronosd: str, group: str, global_seq: int, cfg: dict):
         asyncio.run(send_txs(txs))
 
     # node quit when the chain is idle or halted for a while
-    detect_idle_halted(20, 20)
+    if group == VALIDATOR_GROUP or (group == FULLNODE_GROUP and not exit_on_finish):
```

Comment:
the problem with early quit is it'll lose lots of transactions that are in the fullnode's mempool but not propagated to the validator yet.

Reply:
yea, quite tricky in local since validator also slows down when fullnode survives

Reply:
with validator-generate-load, can you handle a single full node, or you want all full nodes gone? currently only a single full node is required for generating genesis, but that can be changed to validator too.

Reply:
I think so if you mean allow no fullnode as well

Reply:
we can certainly do that